core/src/main/resources/migration/4.0/2017-06-19-OC-8195.xml

```diff
@@ -0,0 +1,490 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+	<changeSet author="jkeremian" id="2017-06-19-OC-8195-01"
+		dbms="postgresql" runOnChange="true">
+		<comment>Before Changing data type of description field in discrepancy_note, we need to drop all Views</comment>
+		<sql>
+			DROP VIEW IF EXISTS view_site_hidden_event_definition_crf;
+			DROP VIEW IF
+			EXISTS view_study_hidden_event_definition_crf;
+			DROP VIEW IF EXISTS view_discrepancy_note;
+			DROP VIEW IF EXISTS
+			view_dn_item_data;
+			DROP VIEW IF EXISTS view_dn_event_crf;
+			DROP VIEW public.view_item_data_toolkit_filtered;
+			DROP VIEW
+			public.view_item_data_toolkit;
+
+		</sql>
+		<rollback>
+			<sql></sql>
+		</rollback>
+	</changeSet>
+
+
+
+
+	<changeSet author="jkeremian" id="2017-06-19-OC-8195-02"
+		dbms="postgresql" runOnChange="true">
+		<comment>Generate dropped Views after updating crf_versions into form_layouts</comment>
+		<sql>
```

Comment:
Can you please replace this with the Liquibase syntax?
 <createView fullDefinition="false" viewName="view_dn_item_data"> </...>

Example
<createView fullDefinition="false" viewName="view_dn_item_data"> SELECT s.study_id,
            s.parent_study_id,
            ( SELECT event_definition_crf.hide_crf
            FROM event_definition_crf
            WHERE ((event_definition_crf.study_event_definition_id = sed.study_event_definition_id) AND ((event_definition_crf.study_id = s.study_id) OR (event_definition_crf.study_id = s.parent_study_id)) AND (event_definition_crf.crf_id = c.crf_id) AND ((event_definition_crf.parent_id = 0) OR (event_definition_crf.parent_id IS NULL)))) AS study_hide_crf,
            ( SELECT event_definition_crf.hide_crf
            FROM event_definition_crf
            WHERE ((event_definition_crf.study_event_definition_id = sed.study_event_definition_id) AND (event_definition_crf.study_id = s.study_id) AND (event_definition_crf.crf_id = c.crf_id) AND ((event_definition_crf.parent_id &lt;&gt; 0) OR (event_definition_crf.parent_id IS NOT NULL)))) AS site_hide_crf,
            dn.discrepancy_note_id,
            map.item_data_id AS entity_id,
            map.column_name,
            ss.study_subject_id,
            ss.label,
            ss.status_id AS ss_status_id,
            dn.discrepancy_note_type_id,
            dn.resolution_status_id,
            s.unique_identifier AS site_id,
            ds.date_created,
            ds.date_updated,
            ds.days,
            ds.age,
            sed.name AS event_name,
            se.date_start,
            c.name AS crf_name,
            ec.status_id,
            i.item_id,
            i.name AS entity_name,
            id.value,
            dn.entity_type,
            dn.description,
            dn.detailed_notes,
            ds.total_notes,
            ua.first_name,
            ua.last_name,
            ua.user_name,
            ua2.first_name AS owner_first_name,
            ua2.last_name AS owner_last_name,
            ua2.user_name AS owner_user_name
            FROM (((((((((((((dn_item_data_map map
            JOIN discrepancy_note dn ON (((dn.discrepancy_note_id = map.discrepancy_note_id) AND ((dn.entity_type)::text = 'itemData'::text) AND ((dn.parent_dn_id IS NULL) OR (dn.parent_dn_id = 0)))))
            JOIN view_dn_stats ds ON ((dn.discrepancy_note_id = ds.discrepancy_note_id)))
            JOIN user_account ua2 ON ((dn.owner_id = ua2.user_id)))
            JOIN item_data id ON ((map.item_data_id = id.item_data_id)))
            JOIN item i ON ((id.item_id = i.item_id)))
            JOIN event_crf ec ON ((id.event_crf_id = ec.event_crf_id)))
            JOIN study_event se ON ((ec.study_event_id = se.study_event_id)))
            JOIN crf_version cv ON ((ec.crf_version_id = cv.crf_version_id)))
            JOIN study_event_definition sed ON ((se.study_event_definition_id = sed.study_event_definition_id)))
            JOIN crf c ON ((cv.crf_id = c.crf_id)))
            JOIN study_subject ss ON ((se.study_subject_id = ss.study_subject_id)))
            JOIN study s ON ((ss.study_id = s.study_id)))
            LEFT JOIN user_account ua ON ((dn.assigned_user_id = ua.user_id)))
            WHERE (map.study_subject_id = ss.study_subject_id);</createView>